docs/content/administration/config-cheat-sheet.en-us.md

```diff
@@ -732,6 +732,7 @@ and
 - `ENABLE_HELO`: **true**: Enable HELO operation.
 - `HELO_HOSTNAME`: **(retrieved from system)**: HELO hostname.
 - `FROM`: **_empty_**: Mail from address, RFC 5322. This can be just an email address, or the "Name" \<email@example.com\> format.
+- `RETURN_PATH`: **_empty_**: Set the Return-Path header, to let mail clients reply to a different mail as it was send from. See [RFC 4021](https://www.rfc-editor.org/rfc/rfc4021#page-17) for more details.
```

Comment:
The comment doesn't seem right. "to let mail clients reply to a different mail as it was send from", IIRC "return path" is not for "a different mail to reply"

Reply:
It confused me as well, pushed docs update to remove second sentence.